.github/workflows/test-check-binary.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     name: Test check_binary.sh for Linux CUDA
     uses: pytorch/test-infra/.github/workflows/linux_job_v2.yml@main
     with:
-      runner: linux.4xlarge.nvidia.gpu
+      runner: linux.g4dn.4xlarge.nvidia.gpu
```

Comment:
Went ahead and made this change here, since this test doesn't trigger on `main` (so this PR was a good place to ensure this worked). As far as I can tell, this job would've been failing since we updated the CUDA default version to be 12.8 a little over a month ago.